churner/churner.go

```diff
@@ -179,7 +179,22 @@ func (c *churner) getOperatorsToChurn(ctx context.Context, quorumIDs []uint8, op
 		}
 
 		if uint32(len(operatorStakes[quorumID])) < operatorSetParams.MaxOperatorCount {
```

Comment:
Cab we assume that `operatorSetParams.MaxOperatorCount` > 0? If so, wouldn't this case include the next one?

Reply:
Practically it should be > 0, but not sure if we can make that assumption